config/initializers/devise.rb

```diff
@@ -1,3 +1,18 @@
+require 'devise_remote_user'
+
+DeviseRemoteUser.configure do |config|
+  config.env_key = lambda do |env|
+    if env['REMOTE_USER']
+      "#{env['REMOTE_USER']}@stanford.edu"
+    elsif Rails.env.development? && ENV['REMOTE_USER']
+      ENV['REMOTE_USER']
+    end
+  end
+  config.auto_create = true
+  config.auto_update = true
+  config.attribute_map = { webauth_groups: 'WEBAUTH_LDAPPRIVGROUP' }
```

Comment:
Are these machines blessed for directory access, or is this just here optimistically?